libs/generic-view/models/src/index.ts

```diff
@@ -123,6 +125,7 @@ export default {
   [blockBox.key]: blockBox,
   [blockPlain.key]: blockPlain,
   [blockHeading.key]: blockHeading,
+  [buttonIcon.key]: buttonIcon,
   [buttonText.key]: buttonText,
```

Comment:
⚪ (suggestion)
What do you think about merging buttonText and buttonIcon into a single component with configurable modes? We could also introduce buttonPrimary and buttonSecondary to handle different variants:

Empty text – displays the icon only.
Text without an icon – displays a button with text.
Text with an icon – displays a button with both text and an icon.

Additionally, the button variant (primary or secondary) could be selected as another configuration parameter.

FYI @mkurczewski